frontend/src/app/application-forms/fields/noncommercial-fields.component.spec.ts

```diff
@@ -19,14 +20,15 @@ class TestComponentWrapperComponent {
 describe('noncommercial fields', () => {
   let component: NoncommercialFieldsComponent;
   let fixture: ComponentFixture<TestComponentWrapperComponent>;
+  let testService: TestService;
 
   beforeEach(
     async(() => {
-      TestBed.configureTestingModule({
-        declarations: [NoncommercialFieldsComponent, TestComponentWrapperComponent],
-        providers: [FormBuilder, ApplicationFieldsService],
-        schemas: [CUSTOM_ELEMENTS_SCHEMA, NO_ERRORS_SCHEMA]
-      }).compileComponents();
+      testService = new TestService();
```

Comment:
Can you initialize this testService w/ the configureTestingModule?

Reply:
@lauraGgit we have added a task in #142 to remove `test.service`